Utilities/rXml.h

```diff
@@ -12,19 +14,21 @@ struct rXmlNode
 	std::string GetName();
 	std::string GetAttribute( const std::string &name);
 	std::string GetNodeContent();
+	void *AsVoidPtr();
 
-	void *handle;
+	pugi::xml_node *handle;
 	bool ownPtr;
 };
 
 struct rXmlDocument
 {
 	rXmlDocument();
-	rXmlDocument(const rXmlDocument& other);
-	rXmlDocument &operator=(const rXmlDocument& other);
+	//rXmlDocument(const rXmlDocument& other);
```

Comment:
Why are these now removed? Shouldn't they be declared deleted else copying one might result in a double free?